### ISSUE TITLE
make install_system error with lisgd.c

### ISSUE THREAD

**Issue by klundry** [Dejvino/pinephone-sway-poc#5]
> cd lisgd && git apply ../patches/lisgd.patch
> ../patches/lisgd.patch:18: space before tab in indent.
>                 transform);
> warning: 1 line adds whitespace errors.
> cd lisgd && make
> make[1]: Entering directory '/home/kelbot/pinephone-sway-poc/lisgd'
> lisgd build options:
> CFLAGS   = 
> LDFLAGS  = -linput
> CC       = cc
> cp config.def.h config.h
> cc -c  lisgd.c
> lisgd.c: In function 'die':
> lisgd.c:41:2: warning: format not a string literal and no format arguments [-Wformat-security]
>    41 |  fprintf(stderr, msg);
>       |  ^~~~~~~
> cc -g -o lisgd lisgd.o -linput
> make[1]: Leaving directory '/home/kelbot/pinephone-sway-poc/lisgd'
> cp lisgd/lisgd /usr/local/bin/

**Comment by klundry** [comment]
After rebooting it boots into sway. I'm not sure if it is related to that error and maybe install_system didn't finish but none of the areas in the waybar do anything when I tap them. The only thing I can do is tap the different workspaces in the top left. They seem to work but I can't open the keyboard or anything.

**Comment by klundry** [comment]
It is also refusing to wake up from screen lock but I will wait to report another bug until I know if this is all related or not.

**Comment by chookity-pokk** [comment]
I ran into this same issue using Arch, did you ever find a solution?

**Comment by klundry** [comment]
> I ran into this same issue using Arch, did you ever find a solution?

No, but at this point I've been using sxmo for quite a while now. It's probably a better bet because it runs sway by default now and has a lot of other nice little tweaks.

**Comment by chookity-pokk** [comment]
Fair enough. I didn't want to deal with having to learn how to use SXMO so I just ended up making my own Sway config to make it feel a bit more like a proper phone OS, https://gitlab.com/greenbeast/ppp-sway-dotfiles

**Comment by klundry** [comment]
I put off trying sxmo for a while too because I wasn't real keen on the button presses and gestures but once I gave it a chance it clicked pretty fast. Also, now with the keyboard accessory I pretty much don't need any of that. 

Oh neat, your config looks really nice!
